### DISCUSSION TITLE
Is any option to modify the label size?

### DISCUSSION THREAD

**Discussion by felixmondelo** [marcusolsson/grafana-gantt-panel#43] [Ideas]
Hi,

I need to minimize the label size because is too big and there is information that I don't need. For example:

![image](https://user-images.githubusercontent.com/14011841/123245334-02880500-d4e5-11eb-9de8-90d7c626f17f.png)

I only need to show the values that are in blue and not the others.

Is any possibility to achieve that?

**Comment by marcusolsson** [comment]
Unfortunately not, there's no currently no way to hide only the metadata.

**Reply by felixmondelo** [reply]
It would be very complex to include this functionality? Any workaround?

**Reply by marcusolsson** [reply]
Not at all. Please create a feature request for this if this is important to you 🙏

**Reply by felixmondelo** [reply]
I opened an Issue, is this valid for you?
